changelogs/fragments/ansible-require-utf8.yml

```
@@ -0,0 +1,6 @@
+major_changes:
+  - ansible - At startup the system encoding and locale are checked to verify they are UTF-8.
```

I wouldn't repeat this here since it's in `breaking_changes` also. But if we're keeping it I'd change it:

```suggestion
  - ansible - At startup the filesystem encoding and locale are checked to verify they are UTF-8.
```